### DISCUSSION TITLE
Recaptcha not validating

### DISCUSSION THREAD

**Discussion by MacGuini** [django-recaptcha/django-recaptcha#287] [General]
I was able to get the Recaptcha checkbox to show up on my form. But I'm still able to submit the form without checking the box.

This documentation is lacking severely. Does anyone have a link to more detailed docs or even an example of how to use it? I've heard praises about the docs for this, but can't seem to find them. It was like pulling teeth just to figure out how to get the box to show up.

**Comment by MacGuini** [comment]
Found a source that explains it better 

https://www.geeksforgeeks.org/how-to-add-google-recaptcha-to-django-forms/

**Comment by Stormheg** [comment]
I agree that the docs could be more helpful. PRs that enhance the documentation are welcome!
